macros/non-standardGeneric.ejs

```diff
@@ -50,7 +50,7 @@ switch($0) {
     %><span class="inlineIndicator nonStandard nonStandardInline"><%-str_inline%></span><%
     break;
   case 'header':
-    %><div class="overheadIndicator nonStandard nonStandardHeader">
+    %><div class="blockIndicator nonStandard nonStandardHeader">
```

Comment:
No pages found using this macro:
https://developer.mozilla.org/en-US/search?topic=none&kumascript_macros=non-standardGeneric&locale=*

Reply:
This is used inside `{{Non-standard_Inline}}` and `{{Non-standard_Header}}`